wbooks-api/app/controllers/api/v1/rents_controller.rb

```diff
@@ -0,0 +1,30 @@
+module Api
+  module V1
+    class RentsController < ApiController
+      def index
+        rents = Rent.all
+        render_paginated rents, each_serializer: RentSerializer
+      end
+
+      def show
+        rent = Rent.find(params[:id])
+        render json: rent, serializer: RentSerializer
+      end
+
+      def create
+        rent = Rent.new(rent_params)
```

Comment:
I prefer use https://apidock.com/rails/ActiveRecord/AttributeAssignment/assign_attributes

Reply:
forget it